hack/compute/apiexport.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: apis.kcp.dev/v1alpha1
 kind: APIExport
 metadata:
-  name: compute-operator
+  name: compute-apis
```

Comment:
I have to change my branch to use that name again.

Reply:
This change was needed to address a mismatch with APIBinding and ClusterRole.